docs/contributing/mbed_os_targets.md

```diff
@@ -56,6 +56,8 @@ The next step to port a target is to enable the test harness dependencies. To ru
 
 Implement the api declared in `mbed-os/hal/gpio_api.h`. You must define the struct `gpio_t`. This struct is commonly defined in an `objects.h` file within the `mbed-os/targets/TARGET_VENDOR/`, `mbed-os/targets/TARGET_VENDOR/TARGET_MCU_FAMILY` or `mbed-os/targets/TARGET_VENDOR/TARGET_MCU_FAMILY/TARGET_MCUNAME` directories.
 
+You should define the Physical LEDs and switches in the target's `PinNames.h` file. LED and switch names are 1-indexed and follow the convention `LED1...LEDn` and `BUTTON1...BUTTONn`. You can see an example [here](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/TARGET_FRDM/PinNames.h#L198).
```

Comment:
Begin incrementing from 1 in this case rather than 0. I wanted to point that out as it's more common to index from 0.

I don't think it has any additional dependencies, I would swing it by someone who requested the change from https://github.com/ARMmbed/mbed-os/issues/4880 to double check that matches their needs.